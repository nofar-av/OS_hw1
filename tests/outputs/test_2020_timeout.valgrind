==26855== Memcheck, a memory error detector
==26855== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==26855== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==26855== Command: /home/omer/OS_wet1/smash
==26855== Parent PID: 26853
==26855== 
==26855== 
==26855== FILE DESCRIPTORS: 4 open at exit.
==26855== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_2020_timeout.valgrind
==26855==    <inherited from parent>
==26855== 
==26855== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_2020_timeout.err
==26855==    <inherited from parent>
==26855== 
==26855== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_2020_timeout.out
==26855==    <inherited from parent>
==26855== 
==26855== Open file descriptor 0:
==26855==    <inherited from parent>
==26855== 
==26855== 
==26855== HEAP SUMMARY:
==26855==     in use at exit: 0 bytes in 0 blocks
==26855==   total heap usage: 582 allocs, 582 frees, 104,508 bytes allocated
==26855== 
==26855== All heap blocks were freed -- no leaks are possible
==26855== 
==26855== For counts of detected and suppressed errors, rerun with: -v
==26855== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
