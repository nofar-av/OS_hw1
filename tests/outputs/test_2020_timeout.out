smash> smash> smash> smash> smash> timeout_sample_file1.txt
timeout_sample_file2.txt
smash> before
smash> smash: got an alarm
after
smash> 0
end: 2
smash> smash: got an alarm
after
smash> 0
2
smash: got an alarm
smash: timeout 3 ../my_sleep 4 timed out!
smash> smash: got ctrl-C
smash> smash> 0
